bucket/sublime-text.json

```diff
@@ -1,7 +1,7 @@
 {
-    "version": "3.2.2-3211",
+    "version": "4-4107",
     "description": "A sophisticated text editor for code, markup and prose",
-    "homepage": "https://www.sublimetext.com/3",
+    "homepage": "https://www.sublimetext.com/download",
```

Comment:
Does not make any sense to have download subpage as homepage when main page is clearly referring to version 4

![image](https://user-images.githubusercontent.com/13260377/119182141-d5be7900-ba72-11eb-8ec0-4253b33fe3d1.png)

Reply:
So `checkver` should scrape the homepage is what you're saying?

Reply:
Resolved because I've made a change to grab the latest version from their API instead.